docs/manual/manual_dronecan.rst

```diff
@@ -44,11 +44,274 @@ Servo modules do not support DroneCAN, as shown in the table below.
+
+The response message will contain the value of the parameter, information about its default, minimum, and maximum values, and the name of the parameter.
+
+Refer to the `uavcan.protocol.param.GetSet section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#getset>`_ in the specification 
+for more details
+
+uavcan.protocol.RestartNode (Data Type ID = 5)
+###############################################
+This request has one field, its “magic number”. This field is an unsigned 40 bit integer. The magic number is used to identify that this is an intentional restart request, 
+and we have also extended it to allow for two different types of reboots. One magic number performs a normal reboot that simply restarts into the normal application firmware, 
+and the other reboots the device into the ST bootloader, which allows it to be programmed with new firmware. The reboot-into-bootloader can be useful for firmware updates. 
+The numbers for each are:
+
+* Normal Reboot Magic Number = 0xACCE551B1E
+* Reboot to Bootloader Magic Number = 0xBEEFCAFE
+
+The response to this request contains one boolean field. This field will be true if the number received was a valid magic number and the motor is restarting, and false if the 
+number received was not one of the valid numbers.
+
+Refer to the `uavcan.protocol.RestartNode section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#restartnode>`_ in the specification 
+for more details.
+
+uavcan.protocol.file.BeginFirmwareUpdate (Data Type ID = 40)
+#############################################################
+On modules that support DroneCAN firmware updates, this command causes the module to enter into upgrade mode, and begin attempting to firmware update over DroneCAN. 
+The request includes a source node ID and a file path on that source node. The module will attempt to get information on the file from the file server, and then begin reading the data from the upgrade file.
+
+On modules that do not support DroneCAN firmware updates, the module will respond with an INVALID_MODE error code to this request.
+
+Refer to the `uavcan.protocol.file.BeginFirmwareUpdate section of Standard Data Types <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#beginfirmwareupdate>`_ in the 
+specification for more details.
+
+.. _dronecan_configuration_parameters:
+
+Configuration Parameters
+**************************
+
+Node ID
+########
+
+.. table::
+
+	+----------------+----------+
+	| **Name**       | **Type** |
+	+----------------+----------+
+	| uavcan_node_id | Integer  |
+	+----------------+----------+
+
+This parameter defines the Node ID of the module on the UAVCAN network. This ID is how the node identifies itself when sending and receiving messages. No two nodes should have 

```

Comment:
some places in the document you capitalize Node ID, and some you put node ID. Should probably pick one and stick with it